docs/standard/assembly/program.md

```diff
@@ -1,78 +1,66 @@
 ---
 title: "Program with assemblies"
 ms.date: "08/20/2019"
-helpviewer_keywords: 
+helpviewer_keywords:
```

Comment:
This file doesn't add any value -- it's basically just a link farm duplicating TOC entries.  I would consider removing it altogether and promoting TOC nodes under it one level up to the level this one was at.  "Programming with" isn't a particularly useful subgroup anyway -- this whole TOC is about "programming with" assemblies.